File: backend/etterem-frontend/src/app/entity/Ratings.ts
export class Ratings{

  public fullname: String;
  public dates:Date;
  public comments:String;
  public ratingpoint:number;


  constructor(fullname?: String, date?: Date, comments?: String, ratingpoint?: number) {
    this.fullname = fullname;
    this.dates = date;
    this.comments = comments || "";
    this.ratingpoint = ratingpoint;
  }
}

File: backend/src/main/java/szakdoga/backend/app/module/Ratings.java
package szakdoga.backend.app.module;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import java.sql.Timestamp;

@Entity
@Table(name = "RATINGS")
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper = true)
@Data
public class Ratings extends BaseEntity {


    @Column(nullable = false)
    private String fullname;
    @Column(nullable = false)
    private Timestamp dates;
    @Column(nullable = false)
    private String comments;
    @Column
    private int ratingpoint;
}
